app/src/MarketTable/CategoricalMarketTable/index.js

```diff
@@ -29,10 +29,11 @@ const MarketTable = ({
 }) => {
   useEffect(() => {
     resetMarketSelections();
+    console.log("resetMarketSelections categorical");
```

Comment:
Remove please? :)

Reply:
Yes, that's the WIP part. I'm still having isssues reseting this variable after marketMakerAddress change